doc/networking.md

```diff
@@ -36,6 +36,7 @@ Verification of `TranscoderRequest` consists of the following steps:
     * TODO distinguish if an orchestrator is lagging behind and hasn't yet received the block with the job. Might require incorporating the node's **current** block number into the request.
 2. Ensure the orchestrator is assigned to the job.
 3. Ensure the current block falls within the valid range of `CreationBlock` ... `EndBlock` for the job.
+4. Ensure the job is still claimable: check it is within 256 blocks of job creation, or a claim has already been submitted for the job.
 4. Verify that the signature comes from the broadcaster.
```

Comment:
number should be changed to `5. `

Reply:
Fixed

Reply:
@j0sh I thought about that possibility, but was too tired to check it out. I think next time we could set all numbers to `1` - when we have different non-consecutive numbers - it looks like a bug, but all ones will look like something intentional, so it should be ok